content/news/posts/2024/do-you-speak-prestashop.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "PrestaShop DevCon 2023: Do you speak PrestaShop"
```

Comment:
```suggestion
title: "PrestaShop DevCon 2023: Do you speak PrestaShop?"
```

Reply:
we used `PSDevCon 2023:` in the previous article from the conference, we probably should keep the same format

Reply:
As suggested by @Julievrz I made an edit.